homeassistant/components/history/__init__.py

```diff
@@ -335,16 +351,16 @@ def _sorted_states_to_json(
         # in-between only provide the "state" and the
         # "last_changed".
         if not ent_results:
-            ent_results.append(next(group).to_native())
+            ent_results.append(States.to_native(next(group)))
 
         initial_state = ent_results[-1]
         prev_state = ent_results[-1]
         initial_state_count = len(ent_results)
 
         for db_state in group:
-            if ATTR_HIDDEN in db_state.attributes and db_state.to_native().attributes.get(
-                ATTR_HIDDEN, False
-            ):
+            if ATTR_HIDDEN in db_state.attributes and States.to_native(
```

Comment:
Seeing all these `to_native` calls really hurt after seeing the logbook API PRs 👍

Reply:
https://github.com/home-assistant/core/pull/36963

Should be better with this change.